doxia-modules/doxia-module-markdown/src/test/java/org/apache/maven/doxia/module/markdown/MarkdownSinkTest.java

```diff
@@ -144,21 +149,18 @@ protected String getFooterBlock() {
 
     /** {@inheritDoc} */
     protected String getListBlock(String item) {
-        return EOL + EOL + Markup.SPACE + "" + MarkdownMarkup.LIST_UNORDERED_ITEM_START_MARKUP + "" + Markup.SPACE
-                + getEscapedText(item) + EOL + EOL;
+        return MarkdownMarkup.LIST_UNORDERED_ITEM_START_MARKUP + "" + Markup.SPACE + getEscapedText(item) + EOL + EOL;
     }
 
     /** {@inheritDoc} */
     protected String getNumberedListBlock(String item) {
-        return EOL + EOL + Markup.SPACE + ""
-                + MarkdownMarkup.LIST_ORDERED_ITEM_START_MARKUP + ""
-                + Markup.SPACE + getEscapedText(item) + EOL + EOL;
+        return MarkdownMarkup.LIST_ORDERED_ITEM_START_MARKUP + "" + Markup.SPACE + getEscapedText(item) + EOL + EOL;
     }
 
     /** {@inheritDoc} */
     protected String getDefinitionListBlock(String definum, String definition) {
-        return EOL + EOL + Markup.SPACE + "" + Markup.LEFT_SQUARE_BRACKET + definum + Markup.RIGHT_SQUARE_BRACKET + ""
-                + Markup.SPACE + definition + EOL + EOL;
+        return "<dl>" + EOL + "<dt>" + getEscapedText(definum) + "</dt>" + EOL + "<dd>" + getEscapedText(definition)
```

Comment:
Why are litaral tags here?

Reply:
https://dev.to/scottshipp/don-t-use-non-test-constants-in-unit-tests-3ej0, I don't think declaring constants for the test only makes this clearer.

Reply:
You should then add a comment why there are literals for clarity.

Reply:
Done in https://github.com/apache/maven-doxia/pull/146/commits/c49775264627b41daa3b17e90f9d72a0b0d9c218.